src/Features/Core/Portable/MoveStaticMembers/AbstractMoveStaticMembersRefactoringProvider.cs

```diff
@@ -26,8 +28,8 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
-            var memberDeclaration = await GetSelectedNodeAsync(context).ConfigureAwait(false);
-            if (memberDeclaration == null)
+            var selectedMemberNodes = await GetSelectedNodesAsync(context).ConfigureAwait(false);
+            if (selectedMemberNodes == null)
```

Comment:
```suggestion
            if (selectedMemberNodes.IsDefaultOrEmpty)
```

(or .IsEmpty).  Just chekcing `== null` is only checking if it's a default ImmutableArrary.  I assume if it's empty you want to bail out here?